src/analyzer/pil_analyzer.rs

```diff
@@ -602,15 +602,16 @@ impl PILContext {
                 BinaryOperator::Mul => left * right,
                 BinaryOperator::Div => left / right,
```

Comment:
I think this should be truncating division on integers and not division in the field.

Reply:
Will update